packages/cli/src/__tests__/artificial-panic.test.ts

```diff
@@ -134,14 +134,14 @@ describe('artificial-panic get-config', () => {
   })
 })
 
-describe('artificial-panic get-dmmf', () => {
+describe('artificial-panic validate', () => {
```

Comment:
Do we also have an artificial panic test for getDMMF somewhere else?

Reply:
You're right that with that test replacement, the artificial panic logic for `getDMMF` was no longer tested. I've added it separately now.